source/features/embed-gist-inline.tsx

```diff
@@ -20,8 +19,11 @@ async function embedGist(link: HTMLAnchorElement): Promise<void> {
 	link.after(info);
 
 	try {
-		const response = await fetch(`${link.href}.json`);
-		const gistData = await response.json();
+		// Get the gist via background.js due to CORB policies introduced in Chrome 73
+		const gistData = await browser.runtime.sendMessage({
+			action: 'fetch',
+			payload: `${link.href}.json`
```

Comment:
Can we shorten this object to just being `{fetch: 'blah blah json'}`. Splitting payload from the action seems unnecessary (even though that’s how I’ve done it for years)

Reply:
I tried doing this and typescript did not like me. Want me to commit?

Reply:
Also the offical **google** docs say to do a check first so that and sendmessage that is passed does not end up by mistake fetched

Reply:
What kind of check?

What does TypeScript say? It’s the same type, except that instead of calling the property `payload` it calls it `fetch`. That’s it.

Reply:
The problem was that `fetch` is a reserved word.

Reply:
Then [`got`](https://github.com/sindresorhus/got)

Reply:
> The problem was that `fetch` is a reserved word.

Ah the problem is in the destructuring part I suppose